types/externs.d.ts

```diff
@@ -199,7 +199,7 @@ declare global {
       /** The hostname to use for the debugging protocol, if manually connecting. */
       hostname?: string;
       /** The level of logging to enable. */
-      logLevel?: 'silent'|'error'|'info'|'verbose';
+      logLevel?: 'silent'|'error'|'info'|'verbose'|'debug';
```

Comment:
```suggestion
      logLevel?: 'silent'|'error'|'warn'|'info'|'verbose'|'debug';
```

Reply:
better yet can we extract and move this type to lighthouse-logger d.ts so they're consistent? it would highlight the `log` issue we have right now :) 

too bad we don't actually type check lighthouse logger